rholang/src/main/scala/coop/rchain/rholang/interpreter/accounting/Costs.scala

```diff
@@ -47,10 +47,14 @@ trait Costs {
   // + allocates byte array of the same size as `serializedSize`
   // + then it copies all elements of the Par
   def toByteArrayCost[T <: GeneratedMessage](a: T): Cost = Cost(a.serializedSize)
+  //TODO: adjust the cost of size method
+  def sizeMethodCost(size: Int): Cost = Cost(size % 100)
```

Comment:
Why `% 100`? If we need a temporary implementation - why not just size?

Reply:
Cost equal to size of the collection is definitely too much. It is temporary but I felt like that big disproportion between simple `size` method and other things is just too much.

Reply:
I'd feel more comfortable if somebody else chimed in here. To me, this should be just `size`, and even `size  / 100` doesn't seem safe to me.

It might be safe, but under the assumption that other parts of the cost model account for this cost. In such a case, I'd rather not be doing any reasoning about the cost model, because it can't be reasoned about locally.

To me, each part that incurs a computational / other cost should account for the cost itself. This also means that the least common denominator of all costs is the cost of the cheapest operation, and if e.g. reading from rspace shold be O(size) and size should be O(size), but the latter with much lower constant - then the rspace read should have a bigger constant.